gcp/compute/test_firewall_opens_all_ports_to_any.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from gcp.compute.helpers import does_firewall_open_all_ports_to_any, firewall_id
+from gcp.compute.resources import in_use_firewalls
+
+
+@pytest.mark.gcp_compute
+@pytest.mark.parametrize("firewall", in_use_firewalls(), ids=firewall_id)
+def test_firewall_opens_all_ports_to_any(firewall):
+    """
+    This test confirms that there are no firewall
+    rules that allow ingress access to all ports from anywhere.
+
+    A part of CIS 3.1
```

Comment:
nit: should we tag CIS-related tests?

Reply:
I think it's a nice reference, especially for newer (for us) services that have CIS Benchmarks.